setup.cfg

```diff
@@ -47,17 +57,53 @@ minor_tag = [minor]
 fix_tag = [fix]
 branch = master
 
-[options]  # generated by wipac:cicd_setup_builder: 'python_requires', 'packages'
-python_requires = >=3.6, <3.11
+[options]  # generated by wipac:cicd_setup_builder: python_requires, packages
+python_requires = >=3.7, <3.11
 packages = find:
 install_requires =
-
-[options.package_data]  # generated by wipac:cicd_setup_builder: '*'
-* = py.typed
+	wipac-dev-tools
 
 [options.extras_require]
+all =
+	wipac-mqclient[pulsar]
+	wipac-mqclient[rabbitmq]
+	wipac-mqclient[gcp]
+	wipac-mqclient[nats]
```

Comment:
I think this cyclic dependency is wrong (or at least it looks weird), and the actual dependencies should be listed here.

Reply:
I thought so too, but it's recommended: https://stackoverflow.com/questions/61609869/setup-py-setup-cfg-install-all-extras

Reply:
I'm not sure I'd call it officially recommended, but I guess it is supported.  Really would have liked them to get `.[option]` recommended, as that's clearer.

I guess this is OK.